src/main/java/com/squareup/javapoet/JavaFile.java

```diff
@@ -39,11 +41,13 @@
   public final CodeBlock fileComment;
   public final String packageName;
   public final TypeSpec typeSpec;
+  public final String indent;
```

Comment:
Indent isn't a property of the `JavaFile`. It's only a property of the `CodeWriter`. Please remove this!